electron/index.html

```diff
@@ -29,7 +29,26 @@ <h3 class="zoom-panel"><span class="zoom-out zoom glow-on-hover">-</span> / <spa
       <div id="tracker-body">
         <!-- START Game View -->
         <div id="game-deck-list" rv-show="game_in_progress">
-          <h3 class="win-loss" rv-show="showWinLossCounter">Wins: { winCounter } &nbsp;&mdash;&nbsp; Losses: { lossCounter }</h3>
+          <div class="win-loss-group">
+            <div class="win-loss-div" rv-show="showTotalWinLossCounter">
+              <span class="win-loss-label">Total</span>
+              <span class="win-loss-counter">W : { totalWinCounter } &nbsp;&mdash;&nbsp; L : { totalLossCounter }</span>
+            </div>
+            <div class="win-loss-div" rv-show="showDailyTotalWinLossCounter">
+              <span class="win-loss-label">Daily Total</span>
```

Comment:
If I leave my tracker open for 24+ hours, does this value reset on it's own? If not, I think it would be more appropriate to label this a "session" total, rather than a daily. (I am pretty sure it doesn't behave this way, but I also think a "daily" counter invites confusion and that we should avoid that name; does it reset at midnight? in what timezone? or is it 24 hours after it starts? etc)

Any issue changing these references to "session" instead of "daily?"

Reply:
Yeah, I guess "session" does sound better. I did intend to actually put in the option to choose a configurable timer for reset instead of just on relaunch, but I've shelved that as overkill for now....